modules/tinymce/src/plugins/table/test/ts/browser/ui/TableCellDialogTest.ts

```diff
@@ -166,7 +170,7 @@ UnitTest.asynctest('browser.tinymce.plugins.table.TableCellDialogTest', (success
         TableTestUtils.sSetDialogValues(advData, true, generalSelectors),
         TableTestUtils.sClickDialogButton('submit dialog', true),
         tinyApis.sAssertContent(advHtml),
-        sAssertEvents([ 'newcell', 'tablemodified' ]),
+        sAssertEvents([{ type: 'newcell', structure: undefined, style: undefined }, { type: 'tablemodified', structure: true, style: true }]),
```

Comment:
Why are we defining `structure` and `style` for `newcell`, as those are never valid properties? I suspect there's a type incorrect someone that needs fixing instead.

- [ ] Fix incorrect types that are requiring these invalid properties